CHANGELOG.md

```diff
@@ -9,7 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Added `BRep.trim` as in-place variation of `slice`.
+
+### Changed
+
```

Comment:
something seems to have gone wrong here...

Reply:
true. a merge conflict accident